docs/app/blog/[slug]/page.tsx

```
@@ -0,0 +1,156 @@
+			</div>
+			<div className="relative mt-2 flex aspect-video w-full items-center justify-center overflow-hidden rounded-xl text-center">
+				<Image
+					src={post.image}
+					alt={post.headline}
+					fill
+					className="inset-auto! object-cover"
+				/>
+			</div>
+			<div
+				className="prose mt-8"
+				dangerouslySetInnerHTML={{ __html: post.html }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)